### PR TITLE
Update travis for branch 8.0

### PR DESCRIPTION
**Pull request by tafaRU**
This PR is related to https://github.com/OCA/maintainer-quality-tools/issues/194

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
+  - TESTS="1" ODOO_REPO="OCA/OCB"
 
 virtualenv:
   system_site_packages: true
 
 install:
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
   - pip install requests
```
Comment (bwrsandman) [inline .travis.yml]: requests is now part of `travis_install_nightly` installation

Comment (tafaRU) [inline .travis.yml]: @bwrsandman, good to know :wink: Just fixed.

**Comment by bwrsandman** [comment]
:+1:

**Comment by eLBati** [comment]
:+1: thanks

**Pull request merged**